assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -674,7 +674,7 @@ che.workspace.provision.secret.labels=app.kubernetes.io/part-of=che.eclipse.org,
 
 # Plugin is added in case async storage feature will be enabled in workspace config
 # and supported by environment
-che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
+che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/latest
```

Comment:
this is not an image, but the plugin. and nightly exists in plugin registry https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins/eclipse/che-async-pv-plugin. With other plugins here like `che.workspace.devfile.default_editor.plugins` or `che.factory.default_plugins`, we're using `/nighly`, so why it is an issue here?

Reply:
it's an issue because nightly plugin definition uses a 'latest' image https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-async-pv-plugin/nightly/meta.yaml#L26

While other nightlies plugins use a `nightly` tag image https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-machine-exec-plugin/nightly/meta.yaml#L27

so the plug-in should not be called nightly, only latest, but as long as che reference the 'nightly' version (which is a latest and didn't get any changes since 4 months) we can't remove 'nightly' in plug-in registry

Reply:
to me it feels more like `che-async-pv-plugin` issue, that has no released tag. In other words we're now using `:latest` image on production, which can be updated anytime and break stuff. Shouldn't we rather follow other plugins, make proper release tag, put the tag into the `latest.txt` and keep nightly as cutting edge?

Reply:
> In other words we're now using :latest image on production, which can be updated anytime and break stuff. 
> Shouldn't we rather follow other plugins, make proper release tag, put the tag into the latest.txt and keep nightly as cutting edge?

why do nightlies on a component that is not updated since 4months ? It will add extra work on release and produce new images that do not changes.
For plug-in broker that is not updated often, we're using tags.

And in parallel we can still create an alias of the image. But it's no longer an issue on this repository.
For example CRW uses digest of images at build time so even if it's `:latest` it's replaced by a digest

Reply:
on our end we are currently working on making the async storage images UBI based, in order to include this in product, and indeed there are some moving parts with the async-storage release cycle. We can technically release the `0.0.1` version of the plugin, add it to the registry + make it latest and refer to it from che-server.

Reply:
because once someone will need to change anything and will not know about all this, he/she might break things for others production deployments. CRW yes, but what about other live systems, like che.openshift.io? It uses nightly now and I don't like an idea to change an image that is used in live production systems.

Reply:
@ibuziuk that's fine but we can still merge this PR as each release of che has its own plugin registry

so Che 7.24 can have latest (latest being 0.0.1) and Che 7.25 still using latest (but here latest could be 0.0.2)

Reply:
> CRW yes, but what about other live systems, like che.openshift.io? It uses nightly now and I don't like an idea to change an image that is used in live production systems.

it won't change anything
Every che release has its own plugin registry.

Reply:
also I think here the discussion is moving out of the scope of this PR

Reply:
> also I think here the discussion is moving out of the scope of this PR

you're right, sorry. Taking step back, from che-server point of view, it's ok for me to use `latest`. I think using `next`/`nightly` would be better because it's the general practice for similar plugins.But we would need to adapt this plugin release cycle first...
However, I'd like to see the `latest` of this plugin to some fixed image tag, so imho we shold fix that asap in plugin registry.

Reply:
> However, I'd like to see the latest of this plugin to some fixed image tag, so imho we shold fix that asap in plugin registry.

@sparkoo 100% agree on that, I see a tag so could probably copy `:latest` to this `:tag` definition, I will ping Ilya

Reply:
FYI https://github.com/eclipse/che-plugin-registry/pull/771 and https://github.com/eclipse/che/pull/18642

Reply:
@benoitf approved PRs above, taken this into account can we update this PR to point to `0.0.1` instead of `latest` ?

Reply:
@ibuziuk we're in the process to keep only one version in plug-in registry (latest) so there would not be anymore `0.0.1` version in the plug-in registry.

Reply:
Ok, I'm probably missing this part since I do fully understand how only `latest` will work from the compatibiltity perspective

Reply:
each che server is tied to a che plug-in registry and QE/tests ensure that every component of che work together.

about compatibility, it means that if you're using che-server 7.24 it will use latest which is 0.0.1 version but maybe in 7.25 it will still use latest but in that case it may be 0.0.1 or 0.0.2 or 0.0.3